x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_advanced_editor/create_analytics_advanced_editor.tsx

```diff
@@ -137,6 +137,7 @@ export const CreateAnalyticsAdvancedEditor: FC<CreateAnalyticsFormProps> = ({ ac
           </EuiFormRow>
           {advancedEditorMessages.map((advancedEditorMessage, i) => (
             <Fragment key={i}>
+              <EuiSpacer />
```

Comment:
Suggest to move this `EuiSpacer` above the line with `.map()` otherwise it will be applied to every item and will result in two spacers between each `EuiCallout` like this:

![image](https://user-images.githubusercontent.com/230104/64867021-d74c6280-d63c-11e9-8452-76292915f5c2.png)